Frontend/android-native/app/src/main/AndroidManifest.xml

```diff
@@ -57,7 +57,7 @@
 
         <provider
             android:name="androidx.core.content.FileProvider"
-            android:authorities="@string/fileProviderPath"
+            android:authorities="${applicationId}.fileProvider"
```

Comment:
if file provider changed we need to change in all the place where file provider is used